### PR TITLE
fixed docs for Jira, requirements

### PR DESCRIPTION
**Pull request by fradeve**

### CODE REVIEW

**Comment by ralphbean** [comment]
:+1: on the jira docs.  Thanks!

I'm not sure I'd accept the ``requirements.txt`` change though -- not all of those deps are actually required (some people run bugwarrior without jira, some without bugzilla, etc..).

It gets tricky on some platforms too -- I think some of those plugins have conflicting requirements about what other dependencies they require (different versions of the ``requests`` library iirc).

File: bugwarrior/docs/services/jira.rst [hunk 1]
```diff
@@ -44,6 +44,11 @@ could add the following configuration option::
 
     jira.query = assignee = ralph and status != closed and status != resolved
 
+This query needs to be modified accordingly to the literal values of you Jira
```
Comment (coddingtonbear) [inline bugwarrior/docs/services/jira.rst]: Super minor typo here -- `s/you Jira/your Jira/g`.

**Comment by fradeve** [comment]
That's completely fine! Let me modify the PR :)

**Comment by ralphbean** [comment]
:+1:.  Thanks @fradeve!

**Pull request merged**